sql/mysqld.h

```diff
@@ -176,7 +176,13 @@ extern const char *log_output_str;
 extern const char *log_backup_output_str;
 extern char *temporal_current_timestamp;
 extern my_bool vers_force;
-extern my_bool vers_hide;
+enum vers_hide_enum: ulong {
```

Comment:
Btw, `Sys_binlog_format` demonstrates how to detect explicit setting (via `ON_UPDATE` hook).